LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthRequestParams.kt

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.BrowserSwitchOptions
+
+/**
+ * Local payment result information.
+ *
+ * @property request used to create the local payment transaction.
+ * @property approvalUrl used for payment approval.
+ * @property paymentId of the local payment after creation.
+ */
+
+class LocalPaymentAuthRequestParams internal constructor(
+    val request: LocalPaymentRequest,
+    val approvalUrl: String,
+    val paymentId: String
+) {
+    var browserSwitchOptions: BrowserSwitchOptions? = null
```

Comment:
I think this class can be a data class and `browserSwitchOptions` should be restricted to library scope or private if possible